multinet/swagger/workspace_tables.yaml

```diff
@@ -0,0 +1,21 @@
+Retrieve the tables of a single workspace.
+---
+parameters:
+  - $ref: "#/parameters/workspace"
+  - $ref: "#/parameters/table_type"
+
+responses:
+  200:
+    description: A list of table names belonging to the specified workspace
+    schema:
+      type: array
+      items:
+        $ref: "#/definitions/entity_name"
```

Comment:
You use "fizzbin" for both the workspace and the table which is technically allowed but might be confusing in the docs. Could we change one?

Reply:
This is just supposed to be an example that shows names coming back, so I'm not super concerned with whether these would be confusing. It will complicate the specs to have different examples for each thing (workspace, table, graph, node id, edge id) so I'd rather just leave it alone.

Reply:
3ee5250f4708fe9427e2a9a5c1a5d93fe0446cf5